upgrade-monitoring-services.md

```diff
@@ -0,0 +1,111 @@
+
+Execute the following command to upgrade Prometheus:
+
+```bash
+tiup cluster patch <cluster-name> prometheus-{new-version}.tar.gz -R prometheus
+```
+
+After the upgrade, you can go to the home page of the Prometheus server (usually at `http://<Prometheus-server-host-name>:9090`), click **Status** in the top navigation menu, and then open the **Runtime & Build Information** page to check the Prometheus version and confirm whether the upgrade is successful.
+
+## Upgrade Grafana
+
+For better compatibility with TiDB, it is recommended to use the Grafana installation package provided in the TiDB installation package. The version of Grafana in the TiDB installation package is fixed. If you want to use a newer Grafana version, refer to [Grafana Release Notes](https://grafana.com/docs/grafana/latest/whatsnew/) for new features of each version and choose a suitable version for your production environment. You can also consult with PingCAP technical staff for a recommended version.
+
+In the following upgrade steps, you need to download the Grafana installation package of your desired version from the Grafana website, and then use it to create a Grafana package that TiUP can use.
+
+### Step 1. Download a new Grafana installation package from the Grafana website
+
+Download a new installation package from the [Grafana download page](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1) and extract it.
+
+### Step 2. Download the Grafana installation package provided by TiDB
+
+1. Download the `TiDB-community-server` package from the [TiDB download page](https://pingcap.com/product/#SelectProduct) and extract it.
+2. In the extracted files, locate `grafana-v{version}-linux-amd64.tar.gz` and extract it.
+
+    ```bash
+    tar -xzf grafana-v{version}-linux-amd64.tar.gz
+    ```
+
+### Step 3. Create a new Grafana package that TiUP can use
+
+1. Copy the files extracted in [Step 1](#step-1-download-a-new-grafana-installation-package-from-the-grafana-website), and then use the copied files to replace the files in the `./grafana-v{version}-linux-amd64/` directory extracted in [Step 2](#step-2-download-the-grafana-installation-package-provided-by-tidb).
+2. Recompress the `./grafana-v{version}-linux-amd64` directory and name the new compressed package as `grafana-v{new-version}.tar.gz`, where `{new-version}` can be specified according to your need.
+
+    ```bash
+    cd grafana-v{version}-linux-amd64.tar.gz
+    tar -zcvf ../grafana-v{new-version}.tar.gz ./
+    ```
+
+### Step 4. Upgrade Grafana using the newly created Grafana package
+
+Execute the following command to upgrade Grafana:
+
+```bash
+tiup cluster patch <cluster-name> grafana-v{new-version}.tar.gz -R grafana
+
+```
+
+After the upgrade, you can go to the home page of the Grafana server (usually at `http://<Grafana-server-host-name>:3000`), and then check the Grafana version on the page to confirm whether the upgrade is successful.
+
+## Upgrade AlertManager
```

Comment:
Can we add instructions for how to check the alertmanager version after the upgrade?

Reply:
added in https://github.com/pingcap/docs/pull/14523/commits/fa5d462feb511afe08df8d65c146f1133459042d

Reply:
Perfect. Thanks.